packages/example/src/ToneJSTesting/ToneJSExample.tsx

```diff
@@ -0,0 +1,79 @@
+import {useEffect, useState} from 'react';
+import {Audio, continueRender, delayRender} from 'remotion';
+
+// Can we use this? :thinking:
+import { getRemotionEnvironment } from '../../../core/dist/get-environment';
+
+import * as Tone from 'tone';
+import toWav from 'audiobuffer-to-wav';
+
+function uint8ToBase64(buffer: ArrayBuffer) {
+	let binary = '';
+	const bytes = new Uint8Array(buffer);
+	const len = bytes.byteLength;
+	for (let i = 0; i < len; i++) {
+		binary += String.fromCharCode(bytes[i]);
+	}
+	return window.btoa(binary);
+}
+
+export const ToneJSExample: React.FC = () => {
+	const [handle] = useState(() => delayRender());
+	const [audioDataURL, setAudioDataURL] = useState('');
+	const remotionEnv = getRemotionEnvironment();
+
+	const renderAudio = async () => {
+		const buffer = await Tone.Offline(() => {
+			const synth = new Tone.Synth().toDestination();
+			const now = Tone.now()
+			synth.triggerAttackRelease("C4", "8n", now + .5)
+			synth.triggerAttackRelease("E4", "8n", now + 1)
+			synth.triggerAttackRelease("G4", "8n", now + 1.5)
+		}, 5);
+
+		if (remotionEnv !== 'rendering') {
+			const player = new Tone.Player().toDestination();
+			player.buffer = buffer;
+			player.start();
+			return;
+		}
+
+		// This should probably be moved to the Audio component.
```

Comment:
I mean, if `<Audio />` supports an AudioBuffer as source, it should decide whether to play it or render it offline depending on the environment. Maybe the Audio component itself could know about encoding an AudioBuffer to base64 🤔

Reply:
I'm probably doing some extra steps here, converting the `AudioBuffer` to `wav` into a `blob:` seems more straightforward. I'll dig a bit more.